modin/pandas/io.py

```diff
@@ -716,17 +716,17 @@ def to_pickle(
 @enable_logging
 def read_spss(
     path: Union[str, pathlib.Path],
-    usecols: Union[Sequence[str], type(None)] = None,
+    usecols: Sequence[str] | None = None,
```

Comment:
This is Python 3.10 syntax, and our minimum version is 3.8. I suggest using `Optional`:

```suggestion
    usecols: Optional[Sequence[str]] = None,
```

Reply:
You are right, but there are many such places in our code. Something will need to be done about this as well.